django_project/feti/static/feti/css/landing-page.css

```diff
@@ -546,7 +546,7 @@
 .result-title img {
     width: 54px;
     height: auto;
-    margin-right: 20px;
+    margin: 10px 5px 5px 5px;
```

Comment:
It will not center small icon and also break other icon
before:
![selection_027](https://cloud.githubusercontent.com/assets/4530905/25113562/90a69734-2422-11e7-9688-f80cad06c8a2.png)

after:
![selection_028](https://cloud.githubusercontent.com/assets/4530905/25113563/90a71c54-2422-11e7-8fcc-3fc80e0e84b6.png)